log/sequencer.go

```diff
@@ -80,6 +81,7 @@ func createMetrics(mf monitoring.MetricFactory) {
 	quota.InitMetrics(mf)
 	seqBatches = mf.NewCounter("sequencer_batches", "Number of sequencer batch operations", logIDLabel)
 	seqTreeSize = mf.NewGauge("sequencer_tree_size", "Size of Merkle tree", logIDLabel)
+	seqTimestamp = mf.NewGauge("sequencer_timestamp", "Time of last STH in ms since epoch", logIDLabel)
```

Comment:
Naming the new one `sequencer_tree_timestamp` would indicate that they're related.

Doc comment should probably say "last STH produced" or "signed" for clarity. Might also want to document both of those as only valid if master for that tree.

Reply:
Added documentation suffix: "last SLR signed". 

All the metrics in this file are only valid for when master so I've omitted that clarification for consistency.